File: Bachelor/Semester4/Systems_for_Design_and_Implementation/Java/Domain/src/main/java/domain/dtos/TripDTO.java
package domain.dtos;

import java.io.Serializable;
import java.time.LocalDateTime;

public class TripDTO implements Serializable {
    private final String transportFirm;
    private final LocalDateTime departureTime;
    private final double price;
    private final int seats;

    public TripDTO(String transportFirm, LocalDateTime departureTime, double price, int seats) {
        this.transportFirm = transportFirm;
        this.departureTime = departureTime;
        this.price = price;
        this.seats = seats;
    }

    public String getTransportFirm() {
        return transportFirm;
    }

    public LocalDateTime getDepartureTime() {
        return departureTime;
    }

    public double getPrice() {
        return price;
    }

    public int getSeats() {
        return seats;
    }
}

File: Bachelor/Semester4/Systems_for_Design_and_Implementation/C#/ProiectCShMPP/Domain/domain/dtos/TripDTO.cs
using System;

namespace ProiectCShMPP.domain.dtos
{
    [Serializable]
    public class TripDTO
    {
        public String TransportFirm
        {
            get;
            set;
        }

        public DateTime DepartureTime
        {
            get;
            set;
        }

        public double Price
        {
            get;
            set;
        }

        public int Seats
        {
            get;
            set;
        }

        public override string ToString()
        {
            return base.ToString()+" "+TransportFirm+" "+DepartureTime+" "+Price+" "+Seats;
        }
    }
}